models/sipnet/R/write.restart.SIPNET.R

```diff
@@ -18,85 +18,82 @@
 ##' @return NONE
 ##' @export
 ##' 
-write.restart.SIPNET<- function(out.dir, runid, time, settings, analysis.vec,
-                                RENAME = TRUE, variables,
-                                sample_parameters = FALSE,
-                                trait.values = NA, met){
+write.restart.SIPNET<- function(outdir, runid, start.time, stop.time,
+                                settings, new.state,
+                                RENAME = TRUE,
+                                new.params = FALSE,
+                                inputs){
   
   rundir<-settings$host$rundir
+  variables <- colnames(new.state)
  
   if(RENAME == TRUE) {
     file.rename(file.path(outdir,runid,"sipnet.out"),
-                file.path(outdir,runid,paste0("sipnet.out",time)))
+                file.path(outdir,runid,paste0("sipnet.out",strftime(as.Date(start.time)-1,"%Y"))))
```

Comment:
Why is there are -1 in start.time?  Also, we should probably put the full date in the filename so that we're not overwriting if the time step is subannual